games-arcade/vvvvvv/vvvvvv-2.0~git.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A puzzle-platformer wherein you change gravity itself instead of jumping"
+DESCRIPTION="Notice requested by the author: \
+VVVVVV is a commercial game! The author has given special permission \
+to make this Haiku version available for free. If you enjoy the game, \
+please consider purchasing a copy at thelettervsixtim.es. \
+\
+VVVVVV is a hit 2010 indie video game from Super Hexagon creator Terry Cavanagh. \
+Play as Captain Viridian, whose must rescue their crew after a teleporter mishap \
+caused them to be seperated in Dimension VVVVVV."
+HOMEPAGE="https://thelettervsixtim.es"
+COPYRIGHT="Terry Cavanagh 2010-2020"
+LICENSE="VVVVVV Source Code License v1.0"
+REVISION="1"
+srcGitRev="3697487f47c3c4d5cae01c7a5b7e8c4c2212e3c6"
+SOURCE_URI="https://github.com/TerryCavanagh/VVVVVV/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="02c131ceee90f92c72e45aed36cb431705ba03465d3a6f31c302f2dc5be55c7a"
+SOURCE_DIR="VVVVVV-$srcGitRev"
+ARCHITECTURES="x86_64 x86"
+ADDITIONAL_FILES="data.zip"
+
+PROVIDES="
+	vvvvvv = $portVersion
+	"
+REQUIRES="
+	haiku
+	llibsdl2
+	sdl2_mixer
```

Comment:
First of, you need lib:lib* (also check the issue for the SDL libs: https://github.com/haikuports/haikuports/issues/3878)

Reply:
Strangely enough, removing bits like `cmd:` and `devel:` did make the build progress further. Last commit that had them was [this](https://github.com/haikuports/haikuports/pull/4652/commits/230457271a342c15377c45514923200eec215167).

Reply:
Also, was not aware of that issue with the SDL libs. Thanks for linking it!